src/web/app/src/components/DependenciesPage.tsx

```diff
@@ -0,0 +1,66 @@
+import { makeStyles } from '@material-ui/core/styles';
+import useSWR from 'swr';
+
+import DependenciesTable from './DependenciesTable';
+import { dependencyDiscoveryUrl } from '../config';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.background.default,
+    fontFamily: 'Spartan',
+    padding: '1em 0 2em 0',
+    paddingTop: 'env(safe-area-inset-top)',
+    wordWrap: 'break-word',
+    [theme.breakpoints.down(1024)]: {
+      maxWidth: 'none',
+    },
+    '& h1': {
+      color: theme.palette.text.secondary,
+      fontSize: 24,
+      transition: 'color 1s',
+      marginTop: 0,
+    },
+    '& p, blockquote': {
+      color: theme.palette.text.primary,
+      fontSize: 16,
+      padding: 8,
+    },
+  },
+  container: {
+    padding: '2vh 18vw',
+    [theme.breakpoints.down(1024)]: {
+      padding: '2vh 8vw',
+      wordWrap: 'break-word',
+    },
+  },
+}));
+
+const DependenciesPage = () => {
+  const classes = useStyles();
+  const { data: dependencies = [] } = useSWR(
+    `${dependencyDiscoveryUrl}/projects`,
+    async (u: string) => {
+      const fetchDependenciesReq = await fetch(u);
+      const fetchDependenciesJson = await fetchDependenciesReq.json();
+      return fetchDependenciesJson;
+    }
```

Comment:
You can drop it if it's not different from the default `fetcher` in https://github.com/Seneca-CDOT/telescope/blob/0972760e8c029e9d6f063b345744ff62af6fb6be/src/mobile/src/App.jsx#L12
```suggestion
```

Reply:
Dropped, thanks for the suggestion. I didn't know that thing exist.